hazelcast-client/src/test/java/com/hazelcast/client/discovery/AwsCloudDiscoveryTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.discovery;
+
+import com.hazelcast.aws.AwsDiscoveryStrategyFactory;
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.client.spi.properties.ClientProperty;
+import com.hazelcast.config.DiscoveryStrategyConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.SlowTest;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({SlowTest.class, ParallelTest.class})
+public class AwsCloudDiscoveryTest {
+    @Test
+    public void testAwsClient_MemberNonDefaultPortConfig() {
+        final Map<String, Comparable> props = new HashMap<String, Comparable>();
+        props.put("hz-port", "60000");
+        props.put("access-key", System.getenv("AWS_ACCESS_KEY_ID"));
+        props.put("secret-key", System.getenv("AWS_SECRET_ACCESS_KEY"));
+        props.put("tag-key", "aws-test-tag");
+        props.put("tag-value", "aws-tag-value-1");
+        props.put("connection-timeout-seconds", "10");
```

Comment:
I'd use `AwsProperties` enums instead of hardcoding property names.
like `props.put(AwsProperties.ACCESS_KEY.name(), "60000");`